pkg/broker/filter/stats_reporter.go

```diff
@@ -38,23 +38,23 @@ var (
 	// eventCountM is a counter which records the number of events received
 	// by a Trigger.
 	eventCountM = stats.Int64(
-		"event_count",
+		"broker_filter_event_count",
```

Comment:
this will break stackdriver integration... these metrics names are whitelisted in SD so that nobody is charged

Reply:
/hold

Reply:
It is wrong as is as they conflict. Knative does not have knowledge of stack driver whitelists.

Reply:
I guess now it has :)
Metrics are API surface. We cannot just change them because code coverage cannot run...